### PR TITLE
MC-40640: [Documentation] Update documentation for MSI Offline distance calculation

### PR DESCRIPTION
**Pull request by thiaramus**
## Purpose of this pull request

This pull request (PR) is to add clarity to the limitations of the offline distance calculations.

https://jira.corp.magento.com/browse/MC-40640
https://jira.corp.magento.com/browse/MC-40705

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:
- [x] Inventory 

## Affected documentation pages

- https://docs.magento.com/user-guide/catalog/inventory-configure-distance-priority.html

## Links to Magento source code or PRs

### CODE REVIEW

**Comment by dobooth** [comment]
If you allow https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork then it is easier for us to process these PRs.

**Comment by thiaramus** [comment]
> If you allow https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork then it is easier for us to process these PRs.

Hi @dobooth, I think in order for it to work you need write access to our L3 organization merchdocs fork, I can't issue them, but you can request them in `#github-permissions` Slack channel.

File: src/catalog/inventory-configure-distance-priority.md [hunk 1]
```diff
@@ -13,9 +13,13 @@ You have two options for calculating the distance and time to find the closest s
 
 - **Offline Calculation** - Calculates the distance using downloaded and imported geocode data using zip/post codes and GPS coordinates to determine the closest source to the shipping destination address. To configure this option, you may require developer assistance to initially download and import geocodes using command line instructions.
 
+{:.bs-callout-info}
+For multistore website with several countries configure [Default Tax Destination]({% link tax/tax-destination-default.md %}) for each country.
```
Comment (dobooth) [inline src/catalog/inventory-configure-distance-priority.md:17]:
```suggestion
For multi-store website with several countries, configure [Default Tax Destination]({% link tax/tax-destination-default.md %}) for each country.
```

File: src/catalog/inventory-configure-distance-priority.md [hunk 2]
```diff
@@ -94,7 +98,10 @@ The key is from the [Google Maps Platform][1] and should have [Geocoding API][2]
 
 ## Configure offline calculation
 
-Offline calculations use country codes to determine the distance between the shipping destination and source addresses. This option may require developer assistance to configure. You will issue a an Inventory Management CLI command to download and import data from [geonames.org][5].
+Offline calculations use country codes to determine the distance between the shipping destination and source addresses. This option may require developer assistance to configure. You will issue an Inventory Management CLI command to download and import data from [geonames.org][5].
```
Comment (dobooth) [inline src/catalog/inventory-configure-distance-priority.md:101]:
```suggestion
Offline calculations use country codes to determine the distance between the shipping destination and source addresses. This option may require developer assistance to configure. You will need to issue an Inventory Management CLI command to download and import data from [geonames.org][5].
```

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**